web/src/test/java/se/inera/intyg/webcert/web/service/facade/impl/GetUserResourceLinksImplTest.java

```diff
@@ -143,6 +143,27 @@ void shallNotIncludeSignedCertificatesListIfUserIsNotDoctor() {
             final var actualLinks = getUserResourceLinks.get(user);
             ResourceLinkFacadeTestHelper.assertExclude(actualLinks, ResourceLinkTypeDTO.ACCESS_SIGNED_CERTIFICATES_LIST);
         }
+
+        @Test
+        void shallNotIncludeChangeUnitIfOriginIsDjupintegration() {
+            final var user = getUserWithOrigin("DJUPINTEGRATION");
+            final var actualLinks = getUserResourceLinks.get(user);
+            ResourceLinkFacadeTestHelper.assertExclude(actualLinks, ResourceLinkTypeDTO.CHANGE_UNIT);
+        }
+
+        @Test
+        void shallIncludeChangeUnitIfOriginIsNormal() {
+            final var user = getUserWithOriginAndRole("NORMAL", false);
+            final var actualLinks = getUserResourceLinks.get(user);
+            ResourceLinkFacadeTestHelper.assertInclude(actualLinks, ResourceLinkTypeDTO.CHANGE_UNIT);
+        }
+
+        @Test
+        void shallIncludeChangeUnitIfOriginIsUthopp() {
+            final var user = getUserWithOriginAndRole("UTHOPP", false);
```

Comment:
For one test getUserWithOrigin is used and for the others getUserWithOriginAndRole. Is it a reason why the same function is not used for all tests?